src/universal/modules/welcome/components/WelcomeWizardForms/Step2TeamName.js

```diff
@@ -11,6 +11,15 @@ import {nextPage, setWelcomeTeam} from 'universal/modules/welcome/ducks/welcomeD
 import shortid from 'shortid';
 import {cashay} from 'cashay';
 
+export const mutationHandlers = {
```

Comment:
I go back and forth on this. Overall question is, mutation handler closest to site of `cashay.mutation` or on the container? Thinking about it fresh, I think it's _probably_ better on the container (vs. how I wrote it) but I could be easily persuaded the other way round...